storage-account.tf

```diff
@@ -1,5 +1,94 @@
 // Shared and specialised Storage Accounts
 
+locals {
+  mgmt_network_name    = "core-cftptl-intsvc-vnet"
+  mgmt_network_rg_name = "aks-infra-cftptl-intsvc-rg"
+
+  sa_aat_subnets = [
```

Comment:
this needs some conditional to only load this in aat I think.

Reply:
i thought this will `var.env == "aat" ? join(",", local.sa_aat_subnets) `

Reply:
not sure if it'll work but you can try it